### DISCUSSION TITLE
How to make a new Journal style note tree?

### DISCUSSION THREAD

**Discussion by maphew** [TriliumNext/Trilium#6073] [Ideas] [answered]
General usage: how do I make a new _Journal_ type note tree, and then create Day entry nodes with in it? 

Scenario: I want to make an episode list for a podcast. Each episode will have a note for the Air Date and then child notes for show notes, transcripts, reading list, etc.  

I created a note called _Episodes_ and copied the attributes from the Trilium Demo 'Journal' note to it. However any child notes I create are just regular notes. How do I enable new notes in this tree to use the calendar widget  chooser and so on?

**Comment by codeqqby** [comment, marked as answer]
set note properties:   #workspaceCalendarRoot ~child:child:child:child:template=yourCustomizedTemplate

**Reply by maphew** [reply]
Thank you, this was helpful.

What I learned:

My copy was actually working, but I needed to create more levels for the template to take effect. The number of `child:child:...` syntax marks the level of nodes. When I posted the above I had created _"Episodes: 1991"_, but didn't see the extra journal widgets. It wasn't until I created _"Episodes: 1991: 10 October: 14 - Sunday"_ that the widgets appeared.

the [`#workspace...`](https://github.com/zadam/trilium/wiki/Workspace) tag portion says to use a calendar specific to ths project (a.k.a. node tree). In my initial design by copying `#calendarRoot` from the demo I was using the global calendar. This isn't necessarily a bad thing, but workspace is certainly more appropriate for this scenario.
